conf/app.routes

```diff
@@ -60,4 +60,9 @@ GET         /unavailable                                      @uk.gov.hmrc.selfs
 GET         /sign-out                                         @uk.gov.hmrc.selfservicetimetopay.controllers.SelfServiceTimeToPayController.signout(continueUrl: Option[String])
 
 # Map static resources from the /public folder to the /assets URL path
-GET         /assets/*file                                     controllers.Assets.at(path="/public", file)
\ No newline at end of file
+GET         /assets/*file                                     controllers.Assets.at(path="/public", file)
+
+
+#TODO: move them to testOnlyDoNotUseInProd.routes
```

Comment:
Do you mind doing this first? I if you look at https://github.com/hmrc/agent-fi-agent-frontend you can see an example of where we set up a test-only endpoints